### ISSUE TITLE
Crash when using -l flag

### ISSUE THREAD

**Issue by andradei** [ogham/exa#567]
Not all directories do this. But one in particular produces an error when I run `exa -l` (taking the flag works. Error below:

```bash
$  RUST_BACKTRACE=full exa -lah
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)', src/libcore/result.rs:999:5
stack backtrace:
   0:        0x1046bbee3 - std::sys::unix::backtrace::tracing::imp::unwind_backtrace::h9ee514c692b21f1f
   1:        0x1046b04b4 - std::panicking::default_hook::{{closure}}::h965a565a39a5c6a6
   2:        0x1046af7cd - std::panicking::rust_panic_with_hook::hfb66282f4c2ad4bd
   3:        0x1046af3bc - std::panicking::continue_panic_fmt::h968476d35e0218a2
   4:        0x1046b1998 - rust_begin_unwind
   5:        0x1046c4869 - core::panicking::panic_fmt::h9ae3f9e1e367adff
   6:        0x10468c5d2 - core::result::unwrap_failed::h02e6997b0638db69
   7:        0x10467177f - exa::output::table::Table::display::h0126c180bedcd47c
   8:        0x1046763d5 - <F as scoped_threadpool::FnBox>::call_box::hded7cbd175ad9dc1
   9:        0x10469ed8e - std::sys_common::backtrace::__rust_begin_short_backtrace::h0b2895b91c97f366
  10:        0x10469f3fa - core::ops::function::FnOnce::call_once{{vtable.shim}}::h55da96ad0c0b45da
  11:        0x1046bc57c - <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::hf9b88d2d2fe073fd
  12:        0x1046bc356 - std::sys::unix::thread::Thread::new::thread_start::h34c632283a609b53
  13:     0x7fff7ddae2ea - _pthread_body
  14:     0x7fff7ddb1248 - _pthread_start
[1]    96866 abort      RUST_BACKTRACE=full exa -lah
```

**Comment by andradei** [comment]
Related to #558 
Might be related to #526 
Related to #479 
Related to #538

**Comment by ogham** [comment]
The fix for this (#670) has been merged and will go out with the next release.
